examples/README.md

```diff
@@ -13,6 +13,7 @@ This page contains a list of example codes written with Optuna.
 * [Chainer](./chainer_simple.py)
 * [ChainerMN](./chainermn_simple.py)
 * [LighGBM](./lightgbm_simple.py)
+* [Mxnet](./mxnet_simple.py)
```

Comment:
Please replace "Mxnet" by "MXNet" (the latter is the official spelling).

Reply:
Okay.